mailbox/plugin/quota-search-elasticsearch/src/test/resources/quotaRatioNoDomain.json

```diff
@@ -0,0 +1 @@
+{"user":"user","domain":null,"quotaRatio":0.55}
```

Comment:
idem

Reply:
Avoid not carefull data formats CF recent PRs and @mbaechler care about this.